frontend/src/components/Patchnotes.jsx

```diff
@@ -20,9 +20,25 @@ const Patchnotes = ({ game }) => {
     },
     comments: [],
   });
+  const [associations, setAssociations] = useState({ nerf: [], buff: [] });
   const [error, setError] = useState(null);
   const [modalOpen, setModalOpen] = useState(false);
   const [currentReply, setCurrentReply] = useState({ message: '', replyToId: null, commentId: null, parentReplyId: null });
+  const [filter, setFilter] = useState('all');
+
+  const classifyUpdate = (text) => {
+    for (let keyword of associations.nerf) {
+      if (text.toLowerCase().includes(keyword[0].toLowerCase()) && text.toLowerCase().includes(keyword[1].toLowerCase())) {
+        return 'nerf';
+      }
+    }
+    for (let keyword of associations.buff) {
+      if (text.toLowerCase().includes(keyword[0].toLowerCase()) && text.toLowerCase().includes(keyword[1].toLowerCase())) {
+        return 'buff';
+      }
+    }
+    return 'neutral';
```

Comment:
nice, isn't it so cool to have a dynamic association like this instead of having to rely on a hard-coded map?